ios/demo/src/MapViewController.m

```diff
@@ -24,8 +24,12 @@ - (void)mapView:(TGMapViewController *)mapView didLoadSceneAsync:(NSString *)sce
     newYork.longitude = -74.00976419448854;
     newYork.latitude = 40.70532700869127;
 
-    [mapView setZoom:16];
-    [mapView setPosition:newYork];
+    TGGeoPoint cairo;
```

Comment:
maybe make sense to keep all demo apps to be centered on manhattan coordinates?

nitpick though!